src/jit/codegenlegacy.cpp

```diff
@@ -22182,9 +22175,9 @@ regNumber          CodeGen::genPInvokeCallProlog(LclVarDsc*            frameList
                               pInfo->inlinedCallFrameInfo.offsetOfCallSiteSP);
 #endif // _TARGET_X86_
 
-    /* mov   dword ptr [frame.callSiteReturnAddress], label */
```

Comment:
this is bad. (the comment example applies to the #else case; nobody added an example for non-xarch codegen)

---
In reply to: [72656662](https://github.com/dotnet/coreclr/pull/6491#discussion_r72656662) [](ancestors = 72656662)